index.js

```diff
@@ -28,22 +28,24 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
```

Comment:
We could also set this to a `const`

Reply:
this was meant to change though, depending on if the option flag was entered

Reply:
hmm... i think when we actually implement the number of results option we might not even need this var... either way is cool with me at this point

Reply:
Scope creep but im cool with it too.